packages/office-ui-fabric-react/src/components/Calendar/CalendarDay.tsx

```diff
@@ -803,12 +806,32 @@ export class CalendarDay extends BaseComponent<ICalendarDayProps, ICalendarDaySt
     return boundedDateRange;
   }
 
+  /**
+   * Returns the index of the first element in the array where the predicate is true, and -1
+   * otherwise.
+   * Note that Array.findIndex() is not supported in IE.
+   * @param items Array of items to be iterated over using the predicate
+   * @param predicate find calls predicate once for each element of the array, in ascending
+   * order, until it finds one where predicate returns true if such an element is found.
+   */
+  private _findIndex<T>(items: T[], predicate: (item: T) => boolean): number {
```

Comment:
Rather than redefine this, would it be useful to use the existing implementation of `findIndex` within the `array.ts` utility?

https://github.com/OfficeDev/office-ui-fabric-react/blob/9a03af36134c277ace2a11d8983d8795cc26a7a6/packages/utilities/src/array.ts#L9

Reply:
Yep, that would be much better.  Thanks for pointing that out.

Reply:
I'll move the findLastIndex as well.

Reply:
Awesome!

Reply:
Actually, I'll move findLastIndex in subsequent different PR.  Looks like moving findLastIndex requires an API review and we need this completed today

Reply:
> 
> 
> Actually, I'll move findLastIndex in subsequent different PR. Looks like moving findLastIndex requires an API review and we need this completed today

Totally acceptable, I'm fine with deferring.